misc/triage/labels.yaml

```diff
@@ -0,0 +1,142 @@
+- name: lifecycle/stale
+  color: "795548"
+  description: Denotes an issue or PR has remained open with no activity and has become stale.
+
+# priority
+- name: priority/awaiting-more-evidence
+  color: fef2c0
+  description: Lowest priority. Possibly useful, but not yet enough support to actually get it done.
+- name: priority/backlog
+  color: fbca04
+  description: Higher priority than priority/awaiting-more-evidence.
+- name: priority/critical-urgent
+  color: e11d21
+  description: Highest priority. Must be actively worked on as someone's top priority right now.
+- name: priority/important-longterm
+  color: eb6420
+  description: Important over the long term, but may not be staffed and/or may need multiple releases to complete.
+- name: priority/important-soon
+  color: eb6420
+  description: Must be staffed and worked on either currently, or very soon, ideally in time for the next release.
+
+# triage
+- name: triage/discuss
+  color: faff84
+  description: Items for discussion
+- name: triage/duplicate
+  color: d455d0
+  from: duplicate
+  description: Indicates an issue is a duplicate of other open issue.
+- name: triage/needs-information
+  color: d455d0
+  description: Indicates an issue needs more information in order to work on it.
+- name: triage/not-reproducible
+  color: d455d0
+  description: Indicates an issue can not be reproduced as described.
+- name: triage/obsolete
+  color: faff84
+  description: Bugs that no longer occur in the latest stable release
+- name: triage/support
+  color: d455d0
+  from: question
+  description: Indicates an issue that is a support question.
+- name: triage/unresolved
+  color: d455d0
+  description: Indicates an issue that can not or will not be resolved.
+
+# component
+- name: co/alpine
+  color: d9ead3
+  description: Issues relating to Alpine image
+- name: co/amazon
+  color: d9ead3
+  description: Issues relating to Amazon Linux image
+- name: co/debian
+  color: d9ead3
+  description: Issues relating to Debian image
+- name: co/oracle
+  color: d9ead3
+  description: Issues relating to Oracle Linux image
+- name: co/photon
+  color: d9ead3
+  description: Issues relating to Photon OS image
+- name: co/redhat
+  color: d9ead3
+  description: Issues relating to RHEL/CentOS image
+- name: co/suse
+  color: d9ead3
+  description: Issues relating to openSUSE/SUSE Linux Enterprise image
+- name: co/ubuntu
+  color: d9ead3
+  description: Issues relating to Ubuntu image
+- name: co/bundler
+  color: d9ead3
+  description: Issues relating to Bundler
+- name: co/cargo
+  color: d9ead3
+  description: Issues relating to Cargo
+- name: co/npm
+  color: d9ead3
+  description: Issues relating to Npm
+- name: co/yarn
+  color: d9ead3
+  description: Issues relating to Yarn
+- name: co/pipenv
+  color: d9ead3
+  description: Issues relating to Pipenv
+- name: co/poetry
+  color: d9ead3
+  description: Issues relating to Poetry
+
+# others
```

Comment:
Should we add one for `docs`? Seems like its a good way to intro people into the project.

Reply:
Do you mean we should add docs/CONTRIBUTING.md? Or, should we refer to it in docs/triage.md?

Reply:
wouldn't that be covered by `kind/documentation`?

Reply:
Ah, I understand. Yes, as Liz said, there is `kind/documentation`.